src/OpenTelemetry/README.md

```diff
@@ -362,7 +365,10 @@ You can also find the exact directory by calling these methods from your code.
    can be an absolute path or a relative path to the current directory.
 
 2. `FileSize` is a positive integer, which specifies the log file size in
-   [KiB](https://en.wikipedia.org/wiki/Kibibyte).
+   [KiB](https://en.wikipedia.org/wiki/Kibibyte). This value must be between 1 MiB
+   and 128 MiB (inclusive), or it will be rounded to the closest upper or lower
+   limit. The log file will never exceed this configured size, and will be
+   rewriten in a circular way.
```

Comment:
```suggestion
   rewritten in a circular way.
```

Reply:
I guess overwritten is more accurate than rewritten?

Reply:
Or just "will be written in a circular way."

Reply:
overwritten sounds good.

Reply:
Updated to overwritten.